public/js/autocomplete.min.js

```diff
@@ -8,4 +8,4 @@
unction(l)&&l.call(e.element,r),e.fixPosition(),d.show(r),n.autoSelectFirst&&(e.selectedIndex=0,r.scrollTop=0,d.addClass(r.firstChild,t)),e.visible=!0,e.findBestHint())):this.hide()},adjustContainerWidth:function(){var e=this.options,t=this.suggestionsContainer;"auto"===e.width&&(e=this.element.offsetWidth-2,t.style.width=(0<e?e:300)+"px")},findBestHint:function(){var n=this.element.value.toLowerCase(),s=null;n&&(this.suggestions.every(function(e){var t=0===e.value.toLowerCase().indexOf(n);return t&&(s=e),!t}),this.signalHint(s))},signalHint:function(e){var t="",n=this;e&&(t=n.currentValue+e.value.substr(n.currentValue.length)),n.hintValue!==t&&(n.hintValue=t,n.hint=e,(this.options.onHint||d.noop)(t))},verifySuggestionsFormat:function(e){return e.length&&"string"==typeof e[0]?e.map(function(e){return{value:e,data:null}}):e},validateOrientation:function(e,t){return e=(e.trim()||"").toLowerCase(),["auto","bottom","top"].includes(e)||(e=t),e},processResponse:function(e,t,n){var s=this,i=s.options;e.suggestions=s.verifySuggestionsFormat(e.suggestions),i.noCache||(s.cachedResponse[n]=e,i.preventBadQueries&&0===e.suggestions.length&&s.badQueries.push(t)),t===s.getQuery(s.currentValue)&&(s.suggestions=e.suggestions,s.suggest())},activate:function(e){"number"!=typeof e&&(e=parseInt(e,10));var t=this.classes.selected,n=this.suggestionsContainer,s=n.getElementsByClassName(this.classes.suggestion)[e];return d.each(n.getElementsByClassName(t),function(e){d.removeClass(e,t)}),this.selectedIndex=e,d.addClass(s,t),s},selectHint:function(){var e=d.inArray(this.hint,this.suggestions);this.select(e)},select:function(e){this.hide(),this.onSelect(e)},moveUp:function(){var e=this;if(-1!==e.selectedIndex)return 0===e.selectedIndex?(d.removeClass(e.suggestionsContainer.children[0],e.classes.selected),e.selectedIndex=-1,e.element.value=e.currentValue,void e.findBestHint()):void e.adjustScroll(e.selectedIndex-1)},moveDown:function(){this.selectedIndex!==this.suggestions.length-1&&this.adjustScroll(this.selectedIndex+1)},adjustScroll:function(e){var t=this,n=t.activate(e),s=n.offsetTop,i=t.suggestionsContainer.scrollTop,o=n.offsetHeight,n=i+t.options.maxHeight-o;s<i?t.suggestionsContainer.scrollTop=s:n<s&&(t.suggestionsContainer.scrollTop=s-t.options.maxHeight+o),t.element.value=t.getValue(t.suggestions[e].value),t.signalHint(null)},onSelect:function(e){var t=this,n=t.options.onSelect,e=t.suggestions[e];t.currentValue=t.getValue(e.value),t.currentValue!==t.element.value&&(t.element.value=t.currentValue),t.signalHint(null),t.suggestions=[],t.selection=e,d.isFunction(n)&&n.call(t.element,e)},getValue:function(e){var t,n=this.options.delimiter;return n?(n=(t=this.currentValue).split(n),t.substr(0,t.length-n[n.length-1].length)+e):e},dispose:function(){var e=this.suggestionsContainer;this.disableKillerFn(),e.parentNode&&e.parentNode.removeChild(e)}},"function"==typeof define&&"object"==typeof define.amd&&define.amd&&define(function(){return l}),e.VanillaAutocomplete=l}(this);
```

Comment:
What was changed here?

Reply:
The default value for auto tagging on a complete match, from `false` to `true`, or more specifically from `!1` to `!0`.

The reason why I changed it in the file itself is because it was like that before I switched it from jQuery to vanilla JavaScript. So to make sure there is at least some parity between the two and do not have to change it everywhere we use the autocomplete.